src/test/base/GovernorBaseTest.sol

```diff
@@ -0,0 +1,54 @@
+pragma solidity ^0.8.9;
+
+import "ds-test/test.sol";
```

Comment:
Will we remove `ds-test` in the future? I saw this issue so I'm imagining that's future work
https://github.com/livepeer/protocol/issues/578

Reply:
correct!